rhessys/util/readtag_worldfile.c

```diff
@@ -89,8 +89,9 @@ param *readtag_worldfile(int *paramCnt, FILE *file,char *key){
 	    if(strcmp(strbuf2,"n_basestations")==0){
 	      break;
 	    }
+	    
```

Comment:
On the other hand, if you have tested this on a reasonably large worldfile (a few GB), and the performance for a 1 day model run is within ~5% of the previous version it might not be worth worrying about.  Up to you.